client/src/api/BackendAccess.tsx

```diff
@@ -41,4 +42,42 @@ export const mutateChore = (object: any) =>
     object: object,
   } as QueryObject);
 
-// for Rewards and Users copy the part above this comment --> ....
```

Comment:
So wie ich am Dienstag verstanden habe, wäre es gut wenn wir die Files folgendermassen verteilen würden:
- BackendAccess pro Dashboard (eines für Eltern, Kinder, und Login/Logout)
- queries bleiben im bestehenden api Ordner

Könnten wir dies so noch umsetzen bevor Severin seine Requests ins selbe File schreibt? :)

Reply:
@marZeu8 , @StuderS41 : Ich versuche gerade die Backend-Zugriffe etwas zu ordnen. Werde das in einem separaten Pull-Request handhaben.